### PR TITLE
Update memstats_gnueabil.py to print objects sizes in a sorted manner

### PR DESCRIPTION
**Pull request by youngeunpark**
Change-Id: Ic7d44d03c84e5b2507ac6d2b0af9d5c5d176f493
Signed-off-by: Youngeun Park <youngeun.park@samsung.com>

### CODE REVIEW

**Comment by sunghan-chang** [comment]
@shivgarg Please review this pull request.

File: os/tools/memstats_gnueabil.py [hunk 1]
```diff
@@ -150,6 +150,63 @@ def printLibrarySizes():
...
+def sortPrintLibrarySizes():
+        results = []
+	nfields = len(Required.keys())
+	for r in Required.keys():
+                print "\t"+r,
+        print "\t Total"
+	for l in allmap.keys():
+		sizearr = [0] * nfields
+		for o in allmap[l].keys():
+			if o in allmap[l].keys():
```
Comment (shivgarg) [inline os/tools/memstats_gnueabil.py]: o will always be in allmap[l].keys(). The if statement is always true.

File: os/tools/memstats_gnueabil.py [hunk 2]
```diff
@@ -150,6 +150,63 @@ def printLibrarySizes():
 			print "\t"+str(sizearr[Required.keys().index(r)]),
 		print "\t" + str(librarySize[l]) + "\t" + l
 
+
+def sortByTotal(n):
+        return int(n[len(n)-2])
+
+def sortPrintLibrarySizes():
+        results = []
```
Comment (shivgarg) [inline os/tools/memstats_gnueabil.py]: The indentations of lines is done with spaces and tabs. Some are indented with spaces and some with tabs. Please follow a single strategy.

File: os/tools/memstats_gnueabil.py (hunk 1 above)
Comment (youngeunpark) [inline os/tools/memstats_gnueabil.py]: yes, it is redundant. let me remove the line 166.

File: os/tools/memstats_gnueabil.py (hunk 2 above)
Comment (youngeunpark) [inline os/tools/memstats_gnueabil.py]: Thx for reviewing it. I will apply a single rule.

**Comment by youngeunpark** [comment]
I replaced this by #119

**Pull request closed without merging**